### ISSUE TITLE
type aliases unsupported

### ISSUE THREAD

**Issue by maxomatic458** [trevyn/turbosql#35]
the following example will not store the data correctly

```rs
type StringType = String;
type Number = i64;


#[derive(Turbosql, Default)]
struct Person {
    rowid: Option<i64>,
    name: Option<StringType>,
    age: Option<Number>,
    image_jpg: Option<Vec<u8>>
}

fn main() -> Result<(), Box<dyn std::error::Error>> {

    let name = "Joe";

    let rowid = Person {
        name: Some(name.to_string()),
        age: Some(42),
        ..Default::default()
    }.insert()?;

    Ok(())
}
```

the name will be stored as the name, but inside quotation marks
the age will be stored as regular text

when trying to query the database for a name, it will not find a match, because it is stored with quotation marks

without type aliases everything works fine

**Comment by trevyn** [comment]
Right, unrecognized arbitrary types are stored JSON-serialized. Unfortunately, as a macro, there's no straightforward way to get underlying type information.

I'm thinking the most reasonable/flexible way to handle this is to allow specifying a different storage type:

```rust
#[derive(Turbosql, Default)]
struct Person {
    rowid: Option<i64>,
    #[turbosql(as = "String")]
    name: Option<StringType>,
    #[turbosql(as = "i64")]
    age: Option<Number>,
    image_jpg: Option<Vec<u8>>
}
```

**Comment by kookyleo** [comment]
> Right, unrecognized arbitrary types are stored JSON-serialized. Unfortunately, as a macro, there's no straightforward way to get underlying type information.
> 
> I'm thinking the most reasonable/flexible way to handle this is to allow specifying a different storage type:
> 
> #[derive(Turbosql, Default)]
> struct Person {
>     rowid: Option<i64>,
>     #[turbosql(as = "String")]
>     name: Option<StringType>,
>     #[turbosql(as = "i64")]
>     age: Option<Number>,
>     image_jpg: Option<Vec<u8>>
> }

Is this writing style currently supported?
